pkg/gateway/gateway.go

```diff
@@ -666,7 +694,8 @@ func (n *jfsObjects) NewMultipartUpload(ctx context.Context, bucket string, obje
 }
 
 const uploadKeyName = "s3-object"
-const partEtag = "s3-etag"
+const s3PartEtag = "s3-part-etag"
```

Comment:
This is not compatible with older version, we should not change that.

Reply:
ok